API.md

```diff
@@ -1034,11 +1034,34 @@ Sets the limit on the maximum number of tabs that the viewer could have at a tim
 
 ```js
 <DocumentView
-  multiTabEnabled={true} // requirement
+  multiTabEnabled={true}
   maxTabCount={5}
 />
 ```
 
+#### onTabChanged
+function, optional
+
+The function is activated when a tab is changed. 
+
+Please note that this API is meant for tab-specific changes. If you would like to know when the document finishes loading instead, see the [`onDocumentLoaded`](#onDocumentLoaded) event.
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+currentTab | string | The title of the current tab
```

Comment:
on android it's the full path i believe

Reply:
On both iOS and Android the full path is returned, so the documentation has been updated